Source/WebCore/platform/CaretAnimator.h

```diff
@@ -90,7 +92,7 @@ class CaretAnimator {
     virtual void setVisible(bool) = 0;
 
     PresentationProperties presentationProperties() const { return m_presentationProperties; }
-    virtual void paint(const Node&, GraphicsContext&, const FloatRect&, const Color&, const LayoutPoint&, const std::optional<VisibleSelection>&) const;
```

Comment:
why is this change being made during upstreaming?

Reply:
good point. will undo